### PR TITLE
doc: update another few mis-handled titles

### PR DESCRIPTION
**Pull request by gvancuts**
After grand update of all titles to use title-case, we found some more
that needed a manual tweak.

Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>

### CODE REVIEW

**Review by dbkinder — CHANGES_REQUESTED** [review]
Thanks Geoffroy. Most changes are fine, some could use an edit...

File: doc/api/GVT-g_api.rst [hunk 1]
```diff
@@ -145,8 +145,8 @@ This section describes the wrap functions:
 
 .. _intel_gvt_ops_interface:
 
-GVT-g Intel_gvt_ops Interface
-*****************************
+GVT-g ``intel_gvt_ops`` Interface
```
[the remark this answers is not in the document]
Comment (gvancuts) [reply doc/api/GVT-g_api.rst]: Makes sense

File: doc/developer-guides/hld/hv-partitionmode.rst [hunk 2]
```diff
@@ -181,14 +181,14 @@ the Service VM startup in sharing mode.
 Inter-Processor Interrupt (IPI) Handling
 ========================================
 
-Guests W/O LAPIC Passthrough
+Guests w/o LAPIC Passthrough
```
[the remark this answers is not in the document]
Comment (gvancuts) [reply doc/developer-guides/hld/hv-partitionmode.rst]: I agree

**Comment by gvancuts** [comment]
I have updated my PR and force-pushed my changes. PTAL

**Review by dbkinder — APPROVED** [review]
LGTM, thanks!

**Comment by fitchbe** [comment]
Just FYI (I can fix this in a new PR): The title-case Python module used the APA style rules, so any word (even a preposition) that's four letters or longer got capitalized. Unless we decide to apply a _different_ style (e.g., Chicago Manual of Style or others that specify _five_ letters or longer), we should just follow the APA guidelines to be consistent and capitalize both _with_ and _without_ in titles.

**Comment by gvancuts** [comment]
> 
> 
> Just FYI (I can fix this in a new PR): The title-case Python module used the APA style rules, so any word (even a preposition) that's four letters or longer got capitalized. Unless we decide to apply a _different_ style (e.g., Chicago Manual of Style or others that specify _five_ letters or longer), we should just follow the APA guidelines to be consistent and capitalize both _with_ and _without_ in titles.

I can do this.

Should we also update these other titles then?
* `vmexit Analysis for ACRN RT Performance` (**for**)
* `Perf/PMU Tools in Performance Analysis` (**in**)
There are some others.

**Comment by fitchbe** [comment]
No, the prepositions _under_ four letters in length should stay lowercase.

**Comment by gvancuts** [comment]
> No, the prepositions _under_ four letters in length should stay lowercase.

Oh yes, I had spaced out on that piece.... I'll update the PR. Thanks!

**Comment by gvancuts** [comment]
> I'll update the PR.

Done, PTAL

**Review by fitchbe — APPROVED** [review]

**Pull request merged**